packages/serve/src/lib/server.ts

```diff
@@ -146,3 +148,11 @@ export class VulcanServer {
       .listen(httpsPort);
   }
 }
+
+// Listen the all uncaught errors (including event emitter errors) to prevent server stop.
+process.on('uncaughtException', async (err) => {
+  // Display non vulcan error
+  if (!(err instanceof VulcanError))
+    logger.warn('Unexpected error happened', err);
+  logger.debug('Make server keep listening');
+});
```

Comment:
We should add the listener when the server started and remove it when the server closed. or everyone who imports anything from `@vulcan-sql/serve` gets this listener.

Reply:
Hi @oscar60310, thanks for suggestion, it's fixed with [a8fe967](https://github.com/Canner/vulcan-sql/pull/109/commits/a8fe9671df3ec9591f5a80ee067f9bb1b2539e77)